js/modules/data.js

```diff
@@ -909,6 +920,11 @@ async function getMessagesByConversation(
   return new MessageCollection(messages);
 }
 
+async function getLastHashBySnode(snode) {
+  const lastHash = await channels.getLastHashBySnode(snode);
```

Comment:
could simplify by simply doing `return channels.getLastHashBySnode(snode);` ?

Reply:
Right I was copying the `getSeenMessagesByHashList` function but I actually wrote that as well haha so I will update them both